corehq/apps/userreports/views.py

```diff
@@ -385,7 +385,7 @@ def import_report(request, domain):
 def report_source_json(request, domain, report_id):
     config = get_document_or_404(ReportConfiguration, domain, report_id)
     config._doc.pop('_rev', None)
-    return json_response(config)
+    return json_response(config._doc)
```

Comment:
i think maybe i'm wrong actually. i thought i had seen that error before but looking closer it seems like i might just be misremembering.